doc/ReactRefresh.md

```diff
@@ -0,0 +1,8 @@
+# React Refresh
+
+- `reagent.dev` ns must be required before anything that loads `react-dom`
+- Don't call `r.dom/render` after reload (e.g. shadow-cljs hook)
+- Call `reagent.dev/refresh!` instead
+- Only components defined using `r/defc` will refresh
+- Reagent doesn't try to create Hook signatures for components,
+  so hook state is reset for updated components.
```

Comment:
I’m curious, if local state will be reset, what’s the motivation to enable react refresh in Reagent?

Reply:
To help projects using both Reagent and UIx/Helix, so they can enable it and get benefit on React Refresh for UIx/Helix components.